### DISCUSSION TITLE
Is there a master page with all cameras?

### DISCUSSION THREAD

**Discussion by jump** [mrlt8/docker-wyze-bridge#344] [Ideas]
I've been poking around and looking at a few things, one thing I would really like (after sound comes out of course) would be a master page that has an "all cameras" view. 

Does that exist already and I'm just not finding it? 

Thank you for this great project.

**Comment by mrlt8** [comment]
This isn't available right now, but could be added in the future. 

I don't think any browsers have native support for RTSP/RTMP and HLS has some latency issues.

Open to suggestions!
